src/suggestion.rs

```diff
@@ -135,6 +145,67 @@ impl<'s> fmt::Display for Suggestion<'s> {
             0
         };
 
+        // We will trim the literal if does not fit in the terminal size
+        // The misspelled word shall always be shown and we aim to show as much
+        // info as possible to the user easily locate the word
+        let terminal_size: usize = get_terminal_size();
+        let mut max_chars: usize = terminal_size;
+        let mut min_chars = 0usize;
```

Comment:
I prefer constants whenever possible.

Reply:
`max_chars` and `min_chars` is also kind of misleading.

Reply:
True, I have changed this code, including many variables names. I hope it is more clear this way, but comments can be also removed if they are too much.

done here: https://github.com/drahnr/cargo-spellcheck/pull/56/commits/e3a10e644baae283357f9667b72b5ca4b8de39e1